source/inference.md

```diff
@@ -645,15 +645,23 @@ glue(
         population_distribution.mark_bar(clip=True).encode(
             x=alt.X(
                 "price",
-                bin=alt.Bin(maxbins=30),
+                bin=alt.Bin(extent=[0, 660], maxbins=40),
                 title="Price per night (Canadian dollars)",
-                scale=alt.Scale(domainMax=700)
+                #scale=alt.Scale(domainMax=700)
             )
         ).properties(
             title='Population', height=150
         ),
-        sample_distribution.properties(title="Sample (n = 40)").properties(height=150),
-        sampling_distribution.properties(
+        sample_distribution.encode(
+            x=alt.X("price")
+                .bin(extent=[0, 660], maxbins=40)
+                .title("Price per night (Canadian dollars)")
+        ).properties(title="Sample (n = 40)").properties(height=150),
+        sampling_distribution.encode(
+            x=alt.X("sample_mean")
+                .bin(extent=[0, 660], maxbins=40)
+                .title("Price per night (Canadian dollars)")
+        ).properties(
```

Comment:
This fix changes the appearance from uneven bins:

![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/4560057/0fff000a-a8b8-4e6d-b87d-d3bd2f11d264)

to even bins:

![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/4560057/ab6e5514-d161-4e7e-9214-fd369b383da2)

which is easier to compare (and more similar to the R version):

![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/4560057/fe18d86b-f5c4-4667-a02c-3c0e90bba4f6)

Reply:
nice catch!